azure-functions/acbs-function/package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "^0.21.1",
     "dotenv": "^8.2.0",
     "durable-functions": "^1.5.2",
-    "moment": "^2.29.1"
+    "moment": "^2.29.1",
+    "date-fns": "^2.25.0"
```

Comment:
doesn't look like is used in acbs-function ?

Reply:
Yep, I use getUnixTime( to get EPOCH from string datetime.